exchanges.yml

```diff
@@ -131,7 +131,11 @@ paths:
         "500":
           description: Internal Error
     post:
-      summary: Participate in an exchange. Posting an empty body will start the exchange or return what the exchange is expecting to complete the next step.
+      summary: Participate in an exchange. Posting an empty body will start the exchange or return
+        what the exchange is expecting to complete the next step. Posting a Verifiable Presentation Request with `query.type`
+        set to "VerifiablePresentationRequest" to start the exchange will result in either a Verifiable Presentation Request 
+        response from the server, if the server will need a Verifiable Presentation from the client during the exchange, or a
+        400 error if the server will issue in response to an empty body.
```

Comment:
> a 400 error if the server will issue in response to an empty body

This last bit is not quite clear, probably just needs more words and a grammar fix.